admin/class-yoast-notification-center.php

```diff
@@ -522,14 +529,30 @@ public function deactivate_hook() {
 	 */
 	public function update_storage() {
 
-		$notifications = $this->get_notifications();
+		$notifications = $this->notifications;
+
+		/**
+		 * @var Yoast_Notification[] $merged_notifications
+		 */
+		$merged_notifications = array();
+		if ( ! empty( $notifications ) ) {
+			$merged_notifications = call_user_func_array( 'array_merge', $notifications );
```

Comment:
Why not doing: `array_merge( $notifications );`

Reply:
`array_merge` expects every array it needs to merge as a separate argument.
I could change it, however, to use argument unpacking. This is a feature [available in PHP >=5.6](https://www.php.net/manual/en/migration56.new-features.php#migration56.new-features.splat):
```
array_merge( ...$notifications );
```

Reply:
We might consider using that, because we now support 5.6 and higher